### PR TITLE
Charting UI Upgrades

### PR DESCRIPTION
**Pull request by honeybadgerofdoom**

### CODE REVIEW

File: src/iframe/js/library/charting/chartBtnNewChartWindow.js [hunk 1]
```diff
@@ -12,34 +12,38 @@ function showGraph() {
 
 function createAddChartArea() {
```
Comment (pierce-smith1) [inline src/iframe/js/library/charting/chartBtnNewChartWindow.js:13]: This function is getting pretty bloated, think about breaking it up

**Review by pierce-smith1 — APPROVED** [review]
Other than that one function everything looks great

File: src/iframe/js/library/charting/chartBtnNewChartWindow.js (hunk 1 above)
Comment (honeybadgerofdoom) [reply src/iframe/js/library/charting/chartBtnNewChartWindow.js:13]: Yeah that's a good idea, I'll do that real quick

**Comment by honeybadgerofdoom** [comment]
@pierce-smith1 I broke that beastie up

**Pull request merged**